tableaudocumentapi/field.py

```diff
@@ -24,6 +24,21 @@
 ]
 
 
+def argument_is_one_of(*allowed_values):
```

Comment:
I think (again, I need to go check after the review, I might be confusing this with TSC) that we have a module specifically for property decorators.  This should probably be in there.  If we don't have one, let's start one :)